src/app/lib/utilities/preprocessor/index.js

```diff
@@ -1,11 +1,12 @@
 import nodeLogger from '../../../helpers/logger.node';
 import applyTimestampRules from './rules/timestamp';
 
-const defaultRules = [applyTimestampRules];
```

Comment:
Hi @ChrisBAshton I had to change how default rules is used here. Do you think default rules and new passed in rules should work like the changes I made? Or was it your intention for new rules to completely override the default rules? If the latter then it would mean I would have to pass in the timestamp rule into https://github.com/bbc/simorgh/pull/1879/files#diff-9927c9cefc48863f54ee5f0b62a72ee4L12

Reply:
@jroebu14 @ChrisBAshton no longer works for the beeb, but he's a really nice guy and might answer anyway - just don't count on it/wait for it :)

Reply:
You could just get rid of default rules, they dont really make sense anyway, as any other page type that doesnt use article data will need to override them anyway.

Line 7 wont work for WS frontpages which cant use the `applyTimestampRules` rule as its a different data format

Reply:
@dr3 OK this makes sense to us. I think we should remove `applyTimestampRules` and use it in `getArticleInitialData`. This makes `preprocessor` more generic too.

Reply:
Sounds good - one side effect is it does change the behaviour for any existing consumers - anyone who called `fetchData` without any preprocessor rules would have previously had the `applyTimestampRules` formatter applied.

If we remove `defaultRules`, they no longer will, but this is likely fine.  `applyTimestampRules` doesn't seem to make sense as a default in the long term